.husky/pre-commit

```diff
@@ -1,3 +1,2 @@
 #!/bin/sh
 pnpm pretty-quick --staged --ignore-path .prettierignore
-pnpm audit
```

Comment:
Welll ... that one, you need to revert ;)

Reply:
done